draft-irtf-cfrg-vdaf.md

```diff
@@ -1322,22 +1328,45 @@ struct {
 } Message;
 ~~~
 
+These messages are used to transition between the states described in
+{{vdaf-prep-comm}, more specifically structured as follows:
+
+~~~
+class State:
+    pass
+
+class Continued(State):
+    def __init__(self, prep_state):
+        self.prep_state = prep_state
+
+class Finished(State):
+    def __init__(self, output_share):
+        self.output_share = output_share
+
+class Rejected(State):
+    def __init__(self):
+        pass
+~~~
+
+Note that there is no representation of the `Start` state as it is never
+instantiated in the ping-pong topology.
```

Comment:
We could just define it as
```python3
class Start(State)
```

Reply:
Whoops, I meant
```python3
class Start(State):
    pass
```

Reply:
I don't want to suggest that implementations should ever do anything about the `Start` state, though. At least with ping-pong, there's no reason for any implementation to ever instantiate a `Start` value or deal with one.